packages/messaging/src/models/token-details-model.ts

```diff
@@ -138,7 +138,7 @@ export class TokenDetailsModel extends DBInterface {
           request.onsuccess = event => {
             const result: TokenDetails | null = (event.target as IDBRequest)
               .result;
-            resolve(result);
+            resolve(result!);
```

Comment:
The line above explicitly says this could be null.

Reply:
Yeah. It is not right. That is why I want to enable --strict, so these issues become obvious :)

I tried not to change any functionality in this PR. I'm already working on fixing the types, and I'll take care of these next week.